### DISCUSSION TITLE
Item gets pinned in-game with CTRL+D

### DISCUSSION THREAD

**Discussion by ledfn** [Kvan7/Exiled-Exchange-2#113] [General]
For me anyway... The default hotkey, CTRL+D pins the item in game. Changing the hotkey to ALT+D does not pin the item.

At first I thought CTRL+D was a new PoE 2 hotkey to pin stuff but it's not, once you change the hotkey to something else, the default hotkey no longer pins the items.

I'm not saying this is a bug, nor a bad thing, but I personally don't like that it pins the item, and I keep hitting CTRL+D because I'm so used to it.. 

Not a huge deal either way, maybe it's a feature people love and enjoy, but I'm just letting you know my experience!

Thanks for reading!

**Comment by Kvan7** [comment]
Yeah this is a bug, I need to see what is going on with it that is causing this. It is most likely that there is a hotkey in game to pin the descriptions. 
Creating an issue since this is something for me to fix

**Reply by Kvan7** [reply]
Should currently be fixed
